src/fights.ts

```diff
@@ -504,6 +564,32 @@ const freeFightSources = [
       ],
     }
   ),
+
+  // Not technically a free fight, but if we haven't delivered a thesis yet, we want to dump it somewhere scaling, and we have Uncle Gator's definitely available.
+  new FreeFight(
+    () => !get("_thesisDelivered") && $familiar`Pocket Professor`.experience >= 400,
```

Comment:
Are we using Prof on our Embezzlers? If so, do we not want to push delivering the thesis until after any prof copies have occured?

Reply:
Per discussion in discord, have made thesis use more aggressive in ef336bb so that we will be back up to full strength by the time we are at embezzlers. Not a perfect compromise, but it'll work